source/documentation/shex/__index.md

```diff
@@ -6,11 +6,8 @@ slug: index
 `jena-shex` is an implementation of the 
 [ShEx (Shape Expressions)](https://shex.io) language.
 
-<p>
-<i>This implementation is experimental, starting with Jena 4.2.0.
-Please send usage reports and experience to </i>
-<tt>users@jena.apache.org</tt>.
-</p>
+> NOTE: This implementation is experimental, starting with Jena 4.2.0.
+> Please send usage reports and experience to <tt>users@jena.apache.org</tt>.
```

Comment:
This can be removed now it's 4.7.0. ShEx is complete.

Reply:
Removed (I left the CSS styles for notes)